tls/s2n_crl.c

```diff
@@ -199,12 +200,13 @@ S2N_RESULT s2n_crl_invoke_lookup_callbacks(struct s2n_connection *conn, struct s
 
     int cert_count = sk_X509_num(validator->cert_chain_from_wire);
     DEFER_CLEANUP(struct s2n_array *crl_lookup_list = s2n_array_new_with_capacity(sizeof(struct s2n_crl_lookup),
-            cert_count), s2n_array_free_p);
+                          cert_count),
+            s2n_array_free_p);
```

Comment:
This wrapping seems wrong

Reply:
See above for the discussion, but here is a alternative:
```
DEFER_CLEANUP(struct s2n_array *crl_lookup_list =
                          s2n_array_new_with_capacity(sizeof(struct s2n_crl_lookup), cert_count),
            s2n_array_free_p);
```